users/file.js

```diff
@@ -0,0 +1,13 @@
+const f = require('fs');
+
+const read = (err,files)=>{
+  let names = '';
+  const find= (element)=>{
+    if(element !=='amey'){
+      names = names + element + '\n';
+    }
+  }
+  files.forEach(find);
+  console.log(names);
```

Comment:
change this to write file. Create a new file using `fs.writeFile(path, data, function)`